contrib/babelfishpg_tds/src/backend/tds/tdsresponse.c

```diff
@@ -660,11 +660,22 @@ resolve_numeric_typmod_from_exp(Node *expr)
 
 			/*
 			 * [BABEL-3074] NUMERIC overflow causes TDS error for aggregate
-			 * function sum(); resultant precision should be individual precision + 1
+			 * function sum(); resultant precision should be tds_default_numeric_precision 
 			 */
 			if (aggFuncName && strlen(aggFuncName) == 3 &&
 					(strncmp(aggFuncName, "sum", 3) == 0))
```

Comment:
I realize this is pre-existing code, but why aren't we just using `strcmp` here instead of having to invite `strlen` to the party as well?